src/Common/Core/Impl/Telemetry/TelemetryServiceBase.cs

```diff
@@ -32,13 +32,13 @@ public abstract class TelemetryServiceBase : ITelemetryService, IDisposable {
         /// </summary>
         public bool IsEnabled {
             get {
-                return this.TelemetryRecorder.IsEnabled;
+                return this.TelemetryRecorder?.IsEnabled == true;
```

Comment:
Ok, let's create a separate task for this. I want to find out, probably another singleton issue.

---
In reply to: [74833030](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74833030) [](ancestors = 74833030,74832919,74832126,74828037)